src/software/proto/play_info_msg.proto

```diff
@@ -0,0 +1,25 @@
+syntax = 'proto3';
+
+
+message PlayInfo
+{
+    message Play
+    {
+        string play_name = 1;
+    }
+
+    message GameState
+    {
+        string referee_command_name = 1;
+    }
+
+    message Tactic
+    {
+        string tactic_name      = 1;
+        string tactic_fsm_state = 2;
```

Comment:
i may have missed it, but I dont see this field being assigned in the AI, so just making sure if that is being left for another ticket?

Reply:
Yeah, we currently don't have anything being assigned to this field, which is also why there is no test for this yet - I will create a new ticket to address assigning the tactic_fsm_state in the AI and testing it